### PR TITLE
addition directories with dlls can be specified now in config file an…

### PR DESCRIPTION
**Pull request by zeus82**
I added the ability to specify additional directories to load dlls from in the config file.

### CODE REVIEW

File: test/TesterInternal/ConfigTests.cs [hunk 1]
```diff
@@ -719,6 +719,20 @@ public void Config_StorageProvider_SomeConfig()
                 }
             }
         }
+
+        [Fact, TestCategory("Functional"), TestCategory("Config"), TestCategory("Providers")]
```
Comment (sergeybykov) [inline test/TesterInternal/ConfigTests.cs]: Why "Providers"?

**Comment by sergeybykov** [comment]
Except for the minor question about "Providers" as a `TestCategory` looks good to me. I kicked off a functional test run, just to make sure nothing breaks due to the slight rearrangements within `SiloAssemblyLoader`.

**Comment by zeus82** [comment]
"Providers" shouldn't be there.. it was a copy and paste mistake.  I have removed it and fixed spelling mistake in test method name.

**Comment by sergeybykov** [comment]
Thank you, @zeus82!

**Pull request merged**